shell.nix

```diff
@@ -0,0 +1,18 @@
+{ withHoogle ? true
+}:
+let
+  default = import ./default.nix {};
+in
+default.nix-tools._raw.shellFor {
+  packages    = ps: with ps; [
+    io-sim
+    io-sim-classes
+    ouroboros-consensus
+    ouroboros-network
+    typed-transitions
+  ];
+  inherit withHoogle;
+  buildInputs = with default.nix-tools._raw; [
+    cabal-install.components.exes.cabal
```

Comment:
To be sure that this package is available through the nix cache, the `shellFor` derivation should probably be included in release.nix. This can be done by moving this `shellFor` in `default.nix` and use [`builds-on-supported-systems`](https://github.com/input-output-hk/cardano-shell/blob/master/release.nix#L13) in `release.nix`.

Reply:
Good idea, thanks!